src/containers/organizations/actions/OrganizationActionTypes.js

```diff
@@ -7,6 +7,8 @@
  */
 
 export const CLEAR_USER_SEARCH_RESULTS :string = 'CLEAR_USER_SEARCH_RESULTS';
+export const DISPLAY_DELETE_MODAL :string = 'DISPLAY_DELETE_MODAL';
```

Comment:
I prefer `SHOW` instead of `DISPLAY` because `SHOW` and `HIDE` are the same character count, and they will align nicely :)

Reply:
Fixed :) And updated the fn to be called showDeleteModal()